### ISSUE TITLE
Nuscenes TestSet Config

### ISSUE THREAD

**Issue by WWW2323** [megvii-research/PETR#10]
Hi, is the [PETRv2 config](https://github.com/megvii-research/PETR/blob/main/projects/configs/petrv2/petrv2_vovnet_gridmask_p4_800x320.py) the corresponding config of PETRv2 in the Nuscenes Leaderboard? Thanks~

**Comment by yingfei1016** [comment]
Hi，you can do the following modifications to reproduce the results of the test set. 
（1）train with the image  size of 640x1600
ida_aug_conf = {
        "resize_lim": (0.94, 1.25),
        "final_dim": (640, 1600),
        "bot_pct_lim": (0.0, 0.0),
        "rot_lim": (0.0, 0.0),
        "H": 900,
        "W": 1600,
        "rand_flip": True,
    }
（2）
num_query=1500,
（3）train with tranval set and cbgs
data = dict(
    samples_per_gpu=1,
    workers_per_gpu=4,
    train=dict(
        type='CBGSDataset',
        dataset=dict(
            type=dataset_type,
            data_root=data_root,
            ann_file=[data_root + 'mmdet3d_nuscenes_30f_infos_train.pkl', data_root + 'mmdet3d_nuscenes_30f_infos_val.pkl'],
            pipeline=train_pipeline,
            classes=class_names,
            modality=input_modality,
            test_mode=False,
            use_valid_flag=True,
            # we use box_type_3d='LiDAR' in kitti and nuscenes dataset
            # and box_type_3d='Depth' in sunrgbd and scannet dataset.
            box_type_3d='LiDAR'),
    ),
    val=dict(type=dataset_type, pipeline=test_pipeline, ann_file=data_root + 'mmdet3d_nuscenes_30f_infos_val.pkl', classes=class_names, modality=input_modality),
    test=dict(type=dataset_type, pipeline=test_pipeline, ann_file=data_root + 'mmdet3d_nuscenes_30f_infos_test.pkl', classes=class_names, modality=input_modality))
（4）pretrain weights of dd3d
load_from='ckpts/dd3d_det_final.pth'

If you have enough memory, you can turn off the checkpoint in [PETRv2 config](https://github.com/megvii-research/PETR/blob/main/projects/configs/petrv2/petrv2_vovnet_gridmask_p4_800x320.py) to speed up.
